diff --git a/Makefile b/Makefile
index 3243b1c..5c1a4e1 100644
--- a/Makefile
+++ b/Makefile
@@ -97,31 +97,24 @@ BIN =\
 	cols\
 	comm\
 	cp\
-	cron\
 	cut\
 	date\
 	dd\
 	dirname\
 	du\
 	echo\
-	ed\
 	env\
 	expand\
 	expr\
 	false\
-	find\
-	flock\
 	fold\
 	getconf\
-	grep\
 	head\
 	hostname\
 	join\
 	kill\
 	link\
 	ln\
-	logger\
-	logname\
 	ls\
 	md5sum\
 	mkdir\
@@ -134,7 +127,6 @@ BIN =\
 	nohup\
 	od\
 	paste\
-	pathchk\
 	printenv\
 	printf\
 	pwd\
@@ -143,7 +135,6 @@ BIN =\
 	rev\
 	rm\
 	rmdir\
-	sed\
 	seq\
 	setsid\
 	sha1sum\
